src/index.html

```diff
@@ -5,8 +5,403 @@
+          <img
+            src="images/imac.jpeg"
+            alt="Monoblock"
+            class="product-card__image"
+          >
+
+          <h2 class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="product-card__code">
+            Product code: 195434
+          </div>
+
+          <div class="product-card__rating">
+            <div class="product-card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="product-card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product-card__price">
+            <div class="product-card__price-text">Price:</div>
+            <div class="product-card__price-amount">$2,199</div>
+          </div>
+
+          <a
+            class="product-card__button product-card__button--hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+
+        <div class="product-card cards-container__product-card">
+          <img
+            src="images/imac.jpeg"
+            alt="Monoblock"
+            class="product-card__image"
+          >
+
+          <h2 class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="product-card__code">
+            Product code: 195434
+          </div>
+
+          <div class="product-card__rating">
+            <div class="product-card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="product-card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product-card__price">
+            <div class="product-card__price-text">Price:</div>
+            <div class="product-card__price-amount">$2,199</div>
+          </div>
+
+          <a
+            class="product-card__button product-card__button--hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+
+        <div class="product-card cards-container__product-card">
+          <img
+            src="images/imac.jpeg"
+            alt="Monoblock"
+            class="product-card__image"
+          >
+
+          <h2 class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="product-card__code">
+            Product code: 195434
+          </div>
+
+          <div class="product-card__rating">
+            <div class="product-card__stars stars stars--4">
```

Comment:
```suggestion
            <div 
              class="
                product-card__stars 
                stars 
                stars--4
              "
            >
```

Reply:
When I change it, npm run lint says error. What should I do?
![photo_2022-09-02_17-52-44](https://user-images.githubusercontent.com/108731612/188192292-261fdfbf-6b1a-49cd-b0f9-97c7ad122f53.jpg)